airbyte-integrations/connectors/destination-azure-blob-storage/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 airbyteBulkConnector {
     core = 'load'
     toolkits = ['load-object-storage', 'load-azure-blob-storage']
-    cdk = '0.428'
+    cdk = 'local'
```

Comment:
Do we want to leave this as local?  I think the team wants this to be pinned.

Reply:
Will pin it when we publish the connector!